recipes/polybeat/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "polybeat" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 17fc46568071985446294b703696697778954f2a8ee62577563e46a6dc3e9a68
+
+build:
+  number: 0
+  skip: true  # [py<3.7]
```

This looks like it could be a noarch python package:
```suggestion
  noarch: python
```